kv/split_test.go

```diff
@@ -169,6 +169,10 @@ func TestRangeSplitsWithWritePressure(t *testing.T) {
 	go startTestWriter(db, int64(0), 1<<15, &wg, nil, nil, done, t)
 
 	// Check that we split 5 times in allotted time.
+	timeout := 3 * time.Second
```

Comment:
I'll make it 6s (10s would run into issues with the default timeout in `make test`).